ground/src/main/java/com/google/android/ground/model/Survey.kt

```diff
@@ -24,6 +24,7 @@ data class Survey(
   val title: String,
   val description: String,
   val jobMap: Map<String, Job>,
+  // TODO(#1730): Allow tile source configuration from a non-survey accessible source.
```

Comment:
It's not immediately clear what we mean by "a non-survey accessible source". Perhaps the TODO here is to clean up / remove `tileSources` from the Survey altogether?

Reply:
Updated TODO.